.github/pull_request_template.md

```diff
@@ -16,25 +20,43 @@
     of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
     description heading and to add a 'x' to the correct box.
 -->
-    [ ] Fix
-    [ ] Refactor
-    [ ] Target update
-    [ ] Functionality change
-    [ ] Docs update
-    [ ] Test update
-    [ ] Breaking change
+    [] Patch update (Bug fix / Target update / Docs update / Test update / Refactor)
+    [] Feature update (New feature / Functionality change / New API)
+    [] Major update (Breaking change E.g. Return code change / API behaviour change)
+
+----------------------------------------------------------------------------------------------------------------
+### Test results (*required*)
```

Comment:
So is it required or optional? It says required in line 28 and optional in 30

Reply:
Required should be updated, I thought I had changed line 30 in the last update - will check